ee/codegen/src/utils/vellum-variables.ts

```diff
@@ -4,6 +4,21 @@ import * as Vellum from "vellum-ai/api";
 import { VELLUM_CLIENT_MODULE_PATH } from "src/constants";
 import { assertUnreachable } from "src/utils/typing";
 
+/**
+ * Creates a built-in list type annotation using lowercase `list[T]` instead of `List[T]` from typing.
+ * This is the modern Python 3.9+ syntax that doesn't require importing from typing.
+ */
+export function builtinListType(itemType: python.Type): python.Type {
+  const listRef = python.reference({
+    name: "list",
+    modulePath: [],
+    genericTypes: [itemType],
+  });
+  const listType = python.Type.reference(listRef);
```

Comment:
DevinAI don't reuse `python.reference`. create a new class that extends `AstNode` called `BuiltinListType`